src/contexts/ComponentAdapter/adapters/ReactAriaComponentAdapter.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ComponentsContextType } from '../ComponentsProvider'
```

Comment:
I'd recommend naming this file something generic like maybe even `DefaultComponentAdapter.tsx`? That way if we moved away from react aria in the future to a diff underlying lib we wouldn't have to rename :D

Reply:
Thats fair, I went back and forth if we should name it or not but i'm game to call it Default